articles/search/search-howto-managed-identities-cosmos-db.md

```diff
@@ -22,10 +22,17 @@ You can use a system-assigned managed identity or a user-assigned managed identi
 
 * [Create a managed identity](search-howto-managed-identities-data-sources.md) for your search service.
 
-* [Assign a role](search-howto-managed-identities-data-sources.md#assign-a-role) in Azure Cosmos DB.
-
+* [Control plane role assignment](search-howto-managed-identities-data-sources.md#assign-a-role) in Azure Cosmos DB.
```

Comment:
For lines 25 through 29, I suggest replacing all of them with the following bullet:

* Assign the [Cosmos DB Account Reader](/azure/role-based-access-control/built-in-roles#cosmos-db-account-reader-role) role to the search service managed identity. This role grants the ability to read Azure Cosmos DB account data. For more information about role assignments in Cosmos DB, see [Configure role-based access control to data](/azure/cosmos-db/how-to-setup-rbac).

Reply:
This is true for Control plane role assignments, but now in cosmosdb we also need to do data plane role assignments mentioned in line 28. which is why i have broken it down into two steps.